packages/cfd/src/Containers/cfd-personal-details-container.tsx

```diff
@@ -123,8 +122,12 @@ const CFDPersonalDetailsContainer = observer(({ onSubmit }: TCFDPersonalDetailsC
 
     return (
         <React.Fragment>
-            <DesktopWrapper>{getPersonalDetailsForm()}</DesktopWrapper>
-            <MobileWrapper>{getPersonalDetailsForm()}</MobileWrapper>
+            <DesktopWrapper>
+                <PersonalDetailsForm />
+            </DesktopWrapper>
+            <MobileWrapper>
+                <PersonalDetailsForm />
+            </MobileWrapper>
```

Comment:
Do we need this twice ? Cant the component itself do the condition ?

Reply:
I think it can be removed because there is no className or mixins used. I know this is not related to your changes @shahzaib-deriv kindly remove it in this PR and check that flow should not be breaking.
Thanks

Reply:
Fair point @thisyahlen-deriv didn't notice that this wrapper is also redundant, 
Thanks @hamza-deriv, will check and update